.travis.yml

```
@@ -20,6 +20,10 @@ matrix:
     - os: osx
       osx_image: xcode9.3
       env: COMPILE_TEST_SNIPPETS=false
+    # JDK 13
+    - os: osx
+      osx_image: xcode11.1
```

```suggestion
      osx_image: xcode11.2
```

Might as well use latest with JDK 13.0.1?